flocker/control/_model.py

```diff
@@ -301,9 +347,8 @@ def __invariant__(self):
         return (True, "")
 
     hostname = field(type=unicode, factory=unicode, mandatory=True)
-    applications = pset_field(Application)
-    manifestations = field(type=PMap, initial=pmap(), factory=pmap,
-                           mandatory=True)
+    applications = pset_field(Application, optional=True)
+    manifestations = pmap_field(unicode, Manifestation, optional=True)
```

Comment:
> We will definitely want them on NodeState, but I was going to do that in followup branch.

Which issue will that be?  FLOC-1570 just says "model objects" which seems to suggest this branch should complete the work.  I don't mind if there's another one to do the rest but maybe update the jira issue to describe the actual intended scope of this work.